specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json

```diff
@@ -272,6 +272,75 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.OperationalInsights/deletedWorkspaces": {
+      "get": {
+        "tags": [
+          "DeletedWorkspaces"
+        ],
+        "x-ms-examples": {
+          "WorkspacesSubscriptionList": {
```

Comment:
I know the payload might be same. Could we change the name of examles?

Reply:
It's the same payload for this version.